deployment/distribution/src/main/kotlin/org/codice/ctk/Command.kt

```diff
@@ -14,141 +14,84 @@ import org.codice.compliance.LENIENT_ERROR_VERIFICATION
 import org.codice.compliance.RUN_DDF_PROFILE
 import org.codice.compliance.TEST_SP_METADATA_PROPERTY
 import org.codice.compliance.USER_LOGIN
-import org.codice.compliance.web.slo.PostSLOTest
-import org.codice.compliance.web.slo.RedirectSLOTest
-import org.codice.compliance.web.slo.error.PostSLOErrorTest
-import org.codice.compliance.web.slo.error.RedirectSLOErrorTest
-import org.codice.compliance.web.sso.PostSSOTest
-import org.codice.compliance.web.sso.RedirectSSOTest
-import org.codice.compliance.web.sso.error.PostSSOErrorTest
-import org.codice.compliance.web.sso.error.RedirectSSOErrorTest
-import org.codice.ctk.Runner.Companion.SLO_BASIC_TESTS
-import org.codice.ctk.Runner.Companion.SLO_ERROR_TESTS
-import org.codice.ctk.Runner.Companion.SSO_BASIC_TESTS
-import org.codice.ctk.Runner.Companion.SSO_ERROR_TESTS
-import org.fusesource.jansi.Ansi.ansi
-import org.junit.platform.engine.TestExecutionResult
-import org.junit.platform.engine.discovery.DiscoverySelectors.selectClass
-import org.junit.platform.launcher.TestExecutionListener
-import org.junit.platform.launcher.TestIdentifier
-import org.junit.platform.launcher.TestPlan
-import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder
-import org.junit.platform.launcher.core.LauncherFactory
-import org.junit.platform.launcher.listeners.SummaryGeneratingListener
 import us.jimschubert.kopper.Parser
 import java.io.File
-import java.io.PrintWriter
-
-private class Runner {
-    companion object {
-        val SSO_BASIC_TESTS = arrayOf(selectClass(PostSSOTest::class.java),
-                selectClass(RedirectSSOTest::class.java))
-        val SSO_ERROR_TESTS = arrayOf(selectClass(RedirectSSOErrorTest::class.java),
-                selectClass(PostSSOErrorTest::class.java))
-        val SLO_BASIC_TESTS = arrayOf(selectClass(PostSLOTest::class.java),
-                selectClass(RedirectSLOTest::class.java))
-        val SLO_ERROR_TESTS = arrayOf(selectClass(PostSLOErrorTest::class.java),
-                selectClass(RedirectSLOErrorTest::class.java))
-    }
-}
 
+@Suppress("StringLiteralDuplication")
 fun main(args: Array<String>) {
     val samlDist = System.getProperty("app.home")
     requireNotNull(samlDist) { "app.home System property must be set" }
 
+    val defaultImplPath = "$samlDist${File.separator}$DEFAULT_IMPLEMENTATION_PATH"
+    val ctkMetadataPath = "$samlDist${File.separator}conf${File.separator}samlconf-sp-metadata.xml"
+
     val parser = createParser()
     val arguments = parser.parse(args)
 
-    val implementationPath = arguments.option("i")
-            ?: "$samlDist${File.separator}$DEFAULT_IMPLEMENTATION_PATH"
+    val implementationPath = arguments.option("i") ?: defaultImplPath
     val userLogin = arguments.option("i") ?: "admin:admin"
```

Comment:
❗️Wasn't modified by this PR but this should be "u"

Reply:
👌